.vib/spring-cloud-dataflow/cypress/cypress/fixtures/imports.json

```diff
@@ -0,0 +1,4 @@
+{
+  "taskStarters": "Task application starters for Docker",
+  "streamStarters": "Stream application starters for RabbitMQ/Docker"
```

Comment:
I don't see where we use here those imports.json. Can we remove RabbitMQ string for another one if it's something to have a string? I don't like mixing product names.

Reply:
This use case is a bit special. Spring CD includes some application starters not imported by default. These imported apps are needed to create tasks and streams, so we must do the import before starting our tests. These 2 strings are just used to select the correct option in the imports section:

<img width="833" alt="Screenshot 2022-09-20 at 18 25 27" src="https://user-images.githubusercontent.com/26460210/191312824-8a6bc23a-c184-4894-8a72-00ccd19ffb47.png">

I could avoid having these strings in a fixture file (maybe as a `const` variable) but I left them there as that's how they were before the revisit.

Reply:
No worries @FraPazGal , I wasn't aware of that detail, so LGTM